code/modules/mob/living/simple_animal/friendly/slime.dm

```diff
@@ -12,42 +12,39 @@
 	response_disarm = "shoos"
 	response_harm   = "stomps on"
 	emote_see = list("jiggles", "bounces in place")
+	holder_type = /obj/item/weapon/holder/animal/slime
 	var/colour = "grey"
-
 	can_butcher = 0
 	meat_type = null
 
 	mob_bump_flag = SLIME
 	mob_swap_flags = MONKEY|SLIME|SIMPLE_ANIMAL
 	mob_push_flags = MONKEY|SLIME|SIMPLE_ANIMAL
 
+/mob/living/simple_animal/slime/attackby(var/obj/item/weapon/slimeparapotion/O as obj, var/mob/user as mob)
+	if(istype(O))
+		canmove = 0
```

Comment:
If you can buckle slimes into chairs, I need to change it. If not, I think it's okay.

I wanted to avoid making a new snowflake var if it was skippable